### PR TITLE
feat(ci): fix message and input

### PR DESCRIPTION
**Pull request by marcosanchotene**
- Add missing line breaks to Mattermost message
- Extract Go version from environment variable to pass it to reusable workflow

### CODE REVIEW

File: .github/workflows/push.yml [hunk 1]
```diff
@@ -172,15 +172,25 @@ jobs:
...
     name: Performance tests
     needs:
       - gosec
       - old-go
+      - go-version
     uses: ./.github/workflows/performance.yml
     secrets: inherit
     with:
-      go-version: ${{ env.GO_VERSION }}
+      go-version: ${{ needs.go-version.outputs.go-version }}
```
Comment (byo) [inline .github/workflows/push.yml:193]: Shouldn't this be:

```suggestion
      go-version: ${{ needs.go-version.outputs.go_version }}
```
?

File: .github/workflows/push.yml [hunk 2]
```diff
@@ -172,15 +172,25 @@ jobs:
           -total-entries-written 300000 \
           -total-entries-read 10000
 
+  go-version:
```
Comment (byo) [inline .github/workflows/push.yml:175]: Can you explain a bit more why this is needed?

File: .github/workflows/push.yml (hunk 1 above)
Comment (marcosanchotene) [reply .github/workflows/push.yml:193]: No. The second `go-version` on line 193 refers to the `output` property of the file on line 179, not to the variable on line 182. I know it's a bit confusing, but I wanted to have an underscore on the variable name.

File: .github/workflows/push.yml (hunk 2 above)
Comment (marcosanchotene) [reply .github/workflows/push.yml:175]: It's a workaround to pass the `$GO_VERSION` environment variable to the called workflow, as this is currently not supported. Please read [this](https://github.com/orgs/community/discussions/26671#discussioncomment-3252793) for reference.

Comment (byo) [reply .github/workflows/push.yml:175]: Ok, got it :+1:.

Can you please add a bit of a comment for this step with the link to mentioned issue?
Looking at that code itself may be confusing for others in the future.

Comment (marcosanchotene) [reply .github/workflows/push.yml:175]: Done, but I added a [link to the official documentation](https://docs.github.com/en/actions/using-workflows/reusing-workflows#limitations).

Comment (byo) [reply .github/workflows/push.yml:175]: Hi @marcosanchotene - I don't see the comment anywhere, can you check if the correct commit was pushed?

Comment (marcosanchotene) [reply .github/workflows/push.yml:178]: Yes, I made a mistake here. It's there now. Sorry!

**Review by byo — APPROVED** [review]

**Review by byo — APPROVED** [review]

**Pull request merged**